src/util/Constants.js

```diff
@@ -512,6 +512,13 @@ exports.ExplicitContentFilterLevels = ['DISABLED', 'MEMBERS_WITHOUT_ROLES', 'ALL
  */
 exports.VerificationLevels = ['NONE', 'LOW', 'MEDIUM', 'HIGH', 'VERY_HIGH'];
 
+/**
+ * The type of membership screening. Here are the available types:
+ * * TERMS
+ * @typedef {string} MembershipScreeningType
+ */
+exports.MembershipScreeningType = ['TERMS'];
```

Comment:
can we add the name here?

maybe
```suggestion
/**
 * The type of membership screening. Here are the available types:
 * * TERMS: Server Rules
 * @typedef {string} MembershipScreeningType
 */
exports.MembershipScreeningType = ['TERMS'];
```

Reply:
Good idea, i'll put `TERMS` in a code block though to make it a little clearer